lib/matplotlib/markers.py

```diff
@@ -216,9 +217,13 @@ def __init__(self, marker=None, fillstyle=None):
         """
         Parameters
         ----------
-        marker : str or array-like or None, default: None
-            *None* means no marker. For other possible marker values see the
-            module docstring `matplotlib.markers`.
+        marker : str, array-like, Path, MarkerStyle, or None, default: None
+
```

Comment:
No need for this first blank line in the description.

Reply:
I wanted to provide a list of explanations for the possible `marker` values.  Do you suggest I use a
```rst
 - bullet list?
```

Reply:
If you want a list, then yes, use a bullet list.